styles/globals.css

```diff
@@ -1,9 +1,31 @@
+:root {
+  --background-color: #2d2e30;
+  --primary-text: #fff;
+  --secondary-text: #ababab;
+  --tertiary-flow-text: #818181;
+  --tertiary-list-text: #16c821;
+  --tertiary-text: #000;
+
+  /* button style */
+  --primary-button: #0bc717;
+  --secondary-button: #109f1c;
+  --tertiary-button: #81e187;
+  --tertiary-button-bg: #d2ffd5;
+
+  /* font-style */
+  --primary-font-size: 45px;
+  --secondary-font-size: 30px;
```

Comment:
This is quite big, is there a reason for it?

Reply:
It´s only the  button font-size.
But it is still in progress.

Reply:
I still wouldn't declare these here and also not in pixels. I'd rather set a "main"-`font-size` and declare these variables with rem